streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/util/FileUtils.java

```diff
@@ -73,4 +82,30 @@ public static byte[] readFileFromOffset(File file, long startOffset, long maxSiz
     }
     return fileContent;
   }
+
+  /**
+   * Roll View Log.
+   *
+   * @param path The file path.
+   * @param offset The offset.
+   * @param limit The limit.
+   * @return The content of the file.
+   * @throws ApiDetailException if there's an error rolling the view log.
+   */
+  public static String rollViewLog(String path, int offset, int limit) {
+    try {
+      File file = new File(path);
+      if (file.exists() && file.isFile()) {
+        try (Stream<String> stream = Files.lines(Paths.get(path))) {
+          List<String> lines = stream.skip(offset).limit(limit).collect(Collectors.toList());
+          StringBuilder builder = new StringBuilder();
+          lines.forEach(line -> builder.append(line).append("\r\n"));
+          return builder.toString();
```

Comment:
The code can be further improved and simplified:
return stream.skip(offset).limit(limit).collect(Collectors.joining("\r\n"));